Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+let countDown;
```

Comment:
Can you come up with a more descriptive name for a variable that stores the return value of `setInterval()`?